src/handlers.ts

```diff
@@ -28,7 +47,8 @@ export const handleRecordChart: Handler = ({ redash, browser }) => {
     const embedUrl = `${redash.alias}/embed/query/${queryId}/visualization/${visualizationId}?api_key=${redash.apiKey}`
     const filename = `${query.name}-${visualization?.name}-query-${queryId}-visualization-${visualizationId}.png`
 
-    const file = await browser.capture(embedUrl)
+    const options = prepareOptions(context.matches.input)
```

Comment:
There are several candidate methods for obtaining input values.
The motivation for retrieving from `matches.input` is to also cover workflow steps, slash command